### ISSUE TITLE
Can't start Pinpoint Daemons

### ISSUE THREAD

**Issue by erinyees** [naver/pinpoint#6846]
- [x] I have checked the [FAQ](http://naver.github.io/pinpoint/faq.html), and [issues](https://github.com/naver/pinpoint/issues) and found no answer.

#### What version of pinpoint are you using?
master

#### Describe your problem**
Can't start PinPoint Daemons on my environment.

#### What have you done?
Steps you have taken to trigger the problem, for example:
I am follow the Quick Start Guide steps but I couldn't it run.

**Case 1. When I hit the command on the Git bash for Windows. - works well**
```
$ sh quickstart/bin/start-collector.sh
---check pinpoint-quickstart-collector process status.---
---initialize pinpoint-quickstart-collector logs.---
rm /c/Users/Hanhg/Desktop/pinpoint/quickstart/logs/quickstart.collector.log.
rm /c/Users/Hanhg/Desktop/pinpoint/quickstart/logs/pid/quickstart.collector.pid.
---pinpoint-quickstart-collector initialization started. pid=1158.---g
starting pinpoint-quickstart-collector. 0 /180 sec(close wait limit).
```

**Case 2. Windows Prompt -- doesn't work**
```
C:\Users\Hanhg\Desktop\pinpoint>quickstart\bin\start-collector.cmd
"Apache Maven (mvn) required."
```

I opened and read the `start-collector.cmd`  and `quickstart.cmd` so I could check it.
1. `mvnw --version`
```
C:\Users\Hanhg\Desktop\pinpoint>mvnw --version
Found "C:\Users\Hanhg\Desktop\pinpoint\.mvn\wrapper\maven-wrapper.jar"
Apache Maven 3.5.3 (3383c37e1f9e9b3bc3df5050c29c8aff9f295297; 2018-02-25T04:49:05+09:00)
Maven home: C:\Users\Hanhg\.m2\wrapper\dists\apache-maven-3.5.3-bin\6rtk6r8hiu511erccgp64thica\apache-maven-3.5.3
Java version: 1.8.0_212, vendor: Azul Systems, Inc.
Java home: C:\Program Files\Zulu\zulu-8\jre
Default locale: ko_KR, platform encoding: MS949
OS name: "windows 10", version: "10.0", arch: "amd64", family: "windows"
```

2. `mvn -version`
```
C:\Users\Hanhg\Desktop\pinpoint>mvn -version
Apache Maven 3.6.3 (cecedd343002696d0abb50b32b541b8a6ba2883f)
Maven home: C:\Users\Hanhg\Downloads\apache-maven-3.6.3\bin\..
Java version: 1.8.0_212, vendor: Azul Systems, Inc., runtime: C:\Program Files\Zulu\zulu-8\jre
Default locale: ko_KR, platform encoding: MS949
OS name: "windows 10", version: "10.0", arch: "amd64", family: "windows"
```

#### Logs
If applicable, please attach agent/collector/web DEBUG log that includes the part where you have encountered the problem. In case of agents, including the start-up log may be of great help.

#### Additional context
Add any other context about the problem here, such as affected library for agents, how your collector/web/hbase is set up if applicable.

My run environment
Windows 10 Enterprise edition
version 1809(OS build 17763.1158)

git --version
`git version 2.26.2.windows.1`

**Comment by koo-taejin** [comment]
hi @erinyees 

This seems to be caused by looking for `mvnw` as a relative path in Windows' command script.

First, if you run the `start-collector.cmd` in the `quickstart/bin` folder, it is expected to operate normally.
Please use in `quickstart/bin` folder until the issue is resolved. 

I'm not good at cmd scripts, so it will take some time, but I will try to fix the issue.

thanks :)

**Comment by erinyees** [comment]
Thanks. @koo-taejin 
It works well!

I typed `quickstart\bin\start-collector.cmd` because the Quick Start Guide guide run like that.
How about additional comment? For example, like below

> Note : If `cmd` command with relative path didn't works on your environment then try below case.
> Move to `${pinpoint root path}\quickstart\bin` and run the `cmd`

**Comment by koo-taejin** [comment]
@erinyees 

Thanks for the good opinion. 

If you send the content as a `pull request`, I will merge it.
If you don't want `pull request`, I'll fix it.

please give me your opinion.
thanks :)

**Comment by erinyees** [comment]
@koo-taejin 
I will. I think I found solution about this problem by searching by the way.


https://stackoverflow.com/questions/14936625/relative-path-in-bat-script
I tested the `%~dp0` and it works but some wrong comment printing both. You can see the state [this link](https://ibb.co/bbFC5wt). If you don't mind can I `pull request` it after fix the `cmd` file command and wrong comment?

**Comment by koo-taejin** [comment]
@erinyees 

I have found out solution at cmd file related with each quickstart
```
for %%i in (%0) do (
   if not defined QUICKSTART_BIN_PATH (
     set QUICKSTART_BIN_PATH=%%~dpi
   )
)
```

In the code above, I have checked that dp is already used for relative paths.

I think this issue can resolve. If I update guide to put system variables for someone who want to execute in other folder

I will close the issue by including the content in the guide.

Thank you for your help. :)

thanks :)
